### PR TITLE
Minor

### LINKED ISSUE #4: Add sunpy intersphinx to conf.py

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #4 as well

### CODE REVIEW

File: {{ cookiecutter.package_name }}/.pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+repos:
```
Comment (Cadair) [inline {{ cookiecutter.package_name }}/.pre-commit-config.yaml]: Can you opt into some of these rather than all of them?

Comment (nabobalis) [reply {{ cookiecutter.package_name }}/.pre-commit-config.yaml]: I don't believe so. We don't have to add all of them.

Comment (nabobalis) [reply {{ cookiecutter.package_name }}/.pre-commit-config.yaml]: How does autopep8 sound instead of black?

Comment (Cadair) [reply {{ cookiecutter.package_name }}/.pre-commit-config.yaml]: stick the whole thing in a cookicutter option, off by default and I am good with it as is.

Comment (nabobalis) [reply {{ cookiecutter.package_name }}/.pre-commit-config.yaml]: I mean it doesn't do anything without being installed manually. 
But I can add an option to copy the file.

File: cookiecutter.json [hunk 2]
```diff
@@ -22,5 +22,6 @@
     "_parent_project": "sunpy",
     "_install_requires": "sunpy",
     "minimum_python_version": ["3.7", "3.6"],
-    "astropy_helpers_version": "v3.2.1"
+    "astropy_helpers_version": "v3.2.1",
+    "use_precommit": "n"
```
Comment (Cadair) [inline cookiecutter.json]:
```suggestion
    "include_precommit_config": "n"
```

File: rendered.yml [hunk 3]
```diff
@@ -20,3 +20,4 @@ default_context:
   sphinx_theme: "sunpy-sphinx-theme"
   initialize_git_repo: "n"
   minimum_python_version: "3.6"
+  use_precommit: "n"
```
Comment (Cadair) [inline rendered.yml]:
```suggestion
  include_precommit_config: "n"
```

File: QUESTIONS.rst [hunk 4]
```diff
@@ -24,3 +24,4 @@ a description of each of the prompts.
 17. ``use_circle_ci``: If 'y' the template will include an example ``.circle/config.yml`` file for the Circle CI service.
 18. ``sphinx_theme``: The value of the ``html_theme`` variable in the sphinx configuration file.
 19. ``minimum_python_version``: Version string of minimum supported Python version
+20. ``use_precommit``: Do you want to use the included pre-commit yaml file.
```
Comment (Cadair) [inline QUESTIONS.rst]:
```suggestion
20. ``include_precommit_config``: Do you want to use the included pre-commit yaml file.
```

File: hooks/post_gen_project.py [hunk 5]
```diff
@@ -65,6 +65,9 @@ def process_licence(licence_name):
     if '{{ cookiecutter.include_example_cython_code }}' != 'y':
         remove_file('{{ cookiecutter.module_name }}/example_c.pyx')
 
+    if '{{ cookiecutter.use_precommit }}' != 'y':
```
Comment (Cadair) [inline hooks/post_gen_project.py]:
```suggestion
    if '{{ cookiecutter.include_precommit_config }}' != 'y':
```

Comment (nabobalis) [inline hooks/post_gen_project.py]:
```suggestion
    if '{{ cookiecutter.include_precommit_config }}' != 'y':
```

**Comment by Cadair** [comment]
CI broke: SAD

**Comment by nabobalis** [comment]
The error is that it can't remove the precommit yaml file. Do I need to add it somewhere else?

**Pull request merged**